config/routes.js

```diff
@@ -184,6 +184,7 @@ module.exports.routes = {
   'POST /api/v1/massifs/:id/subscribe': 'v1/massif/subscribe',
   'POST /api/v1/massifs/:id/unsubscribe': 'v1/massif/unsubscribe',
   'PUT /api/v1/massifs/:id': 'v1/massif/update',
+  'GET /api/v1/massifs/dashboard-data/:id': 'v1/massif/get-dashboard-data',
```

Comment:
Should be : 

`GET /api/v1/massifs/:id/dashboard-data`

In REST API, you first target which resource you need (massif with id = 42 for example), than you request what you want about it (the dashboard data). 

<hr/>

Another thing, the API (and the service DashboardDataMassifService) doesn't know how the data will be used (maybe in a dashboard but maybe not ?): the word "dashboard" is inappropriate here. I think you need to find something else such as "statistics" or "overview" for example 🙂

Reply:
Good point, we will make the changes.